ui/src/overview.tsx

```diff
@@ -0,0 +1,104 @@
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import Box from "@mui/material/Box";
+import Chip from "@mui/material/Chip";
+import Divider from "@mui/material/Divider";
+import Grid from "@mui/material/Grid";
+import Paper from "@mui/material/Paper";
+import Stack from "@mui/material/Stack";
+import Typography from "@mui/material/Typography";
+import {
+  bindMenu,
+  bindTrigger,
+  usePopupState,
+} from "material-ui-popup-state/hooks";
+import React from "react";
+import { Criteria, Group, State } from "./state";
+
+type OverviewProps = {
+  underlay: string;
+  state: State;
+};
+
+export default function Overview(props: OverviewProps) {
+  return (
+    <>
+      <Grid container columns={3}>
+        <Grid item xs={1} sx={{ mx: 2 }} key="included">
```

Comment:
my understanding is that key is only really needed when you have multiple siblings of the same component type generated dynamically. I don't think this usage of key is necessary